modules/external-secrets-operator-uninstall-helm.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * security/external_secrets_operator/external-secrets-operator-migrate-downstream-upstream.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="external-secrets-operator-uninstall-helm_{context}"]
+= Uninstalling a helm installed community {external-secrets-operator-short}
+
+.Procedure
+
+If you installed the community {external-secret-operator} by Helm, delete the Operator by performing the following steps:
+
+. Install the {external-secrets-operator}. The `external-secrets-operator` namespace must be null.
+
+. Delete the {external-secrets-short} by running the following command:
++
+[source,terminal]
+----
+$ oc helm delete <release_name> -n <operator_namespace>
+----
++
+[NOTE]
+====
+Using `helm delete` might delete all Custom Resource Definitions (CRDs) and CRs.
```

Comment:
Using `helm delete` might delete all Custom Resource Definitions (CRDs) and CRs. So recommend to install the downstream operator first if the ns external-secrets-operator is empty.